Runtime/UrdfComponents/UrdfRobot.cs

```diff
@@ -49,6 +50,16 @@ public void ChangeToCorrectedSpace(bool rosSpace)
             this.transform.Rotate(0, 180, 0);
         }
 
+        public bool CheckOrientation()
+        {
+            return currentOrientation == choosenAxis;
+        }
+
+        public void SetOrientation()
```

Comment:
same, this one-line function seems only used in one place. Don't have to be a separated function.